sprints/dashboard/models.py

```diff
@@ -85,7 +85,7 @@ def __init__(
             self.current_sprint = False
         try:
             self.story_points = getattr(issue.fields, issue_fields['Story Points'], None)
-        except (AttributeError, TypeError):
+        except AttributeError:
```

Comment:
We don't need `AttributeError` as well - we're defining the default value here, no?

Reply:
yep